docs/introduction/scoping.md

```diff
@@ -1,29 +1,32 @@
 # Scoping
 
-Scope defines a boundary or context for sets of economic events, commitments, and intents.  This can be for purposes of:
+`vf:inScopeOf`
+
+Scoping allows to group entities together to define a boundary or context. Mostly used for sets of economic events, commitments, and intents.  This can be for purposes of:
+
```

Comment:
Improving the English, suggest:

Scoping allows grouping entities together in a defined boundary or context.